lib/ApplicationFeatures/ApplicationFeature.cpp

```diff
@@ -158,17 +150,16 @@ void ApplicationFeature::addAncestorToAllInPath(
     pathTypes.emplace_back(ancestorType);  // make sure we show the duplicate
 
     // helper for string join
-    std::function<std::string(std::type_index)> cb =
-        [](std::type_index type) -> std::string {
-      return boost::core::demangle(type.name());
```

Comment:
As we are not using `boost::core::demangle` anymore in ApplicationFeature.cpp, we can now remove the include in line 24:
```
#include <boost/core/demangle.hpp>
```